.ci/packer_cache.sh

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env bash
-source /usr/local/bin/bash_standard_lib.sh
+set +e
 
-JAVA_HOME=$HOME/.java/java11 \
-  ./mvnw dependency:go-offline --fail-never -q -B
+source /usr/local/bin/bash_standard_lib.sh
 
-JAVA_HOME=$HOME/.java/java11 ./mvnw clean verify \
-  --fail-never -q -B \
+retry 3 JAVA_HOME=$HOME/.java/java11 \
```

Comment:
Do we have any idea about the root cause of the connection issues? It never happens locally. Could it be an OS or network setup issue?

Reply:
since the issue is easy to replicate in a local environment with an empty .m2 repository, the most probable cause is the implementation of the retry system for downloads that maven have, many downloads at the same time, and the service of maven central, we are not the only people that face the issue.